packages/js/product-editor/src/blocks/pricing/block.json

```diff
@@ -8,7 +8,7 @@
 	"keywords": [ "products", "price" ],
 	"textdomain": "default",
 	"attributes": {
-		"name": {
+		"property": {
```

Comment:
I think that [this 'name'](https://github.com/woocommerce/woocommerce/pull/40521/files#diff-0313866555230023b2954c4de2af62c3ff086977f8f604ec301ae35c622b744aR64) will always be `undefined`.